.github/workflows/linux.yml

```diff
@@ -179,6 +163,30 @@ jobs:
             tar -czvf ${BUILD_DIR}/openvino_tests.tar.gz *
           popd
 
+      - name: Build Debian packages
+        run: |
+          /usr/bin/python3.8 -m pip install -U pip
```

Comment:
Setup Python 3.8 for the whole pipeline? The requirement for the py3.8 is the limitation of the Debian packages build?

Reply:
Good question.
Python 3.8 is required because it's a default version for Ubuntu 20.04. So, we build Debian packages with Python 3.8 and such packages can later be installed using `apt-get install openvino` and automatically install default version of python.

I suppose we can keep as is and return later when we will add all python versions supported by OpenVINO. So, we will decide which version to use by default in pipelines.